src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$secondary-color-gray: #616070;
```

Comment:
You need to fix the variable names, they should show their purpose, not their content. if I decided to change the color, I would have to change it everywhere, not just in the variable definition.